packages/dart/consumer_auth_provider/pubspec.yaml

```diff
@@ -10,7 +10,6 @@ resolution: workspace
 
 dependencies:
   affinidi_tdk_common: ^1.0.0
-  affinidi_tdk_cryptography: ^1.0.0
```

Comment:
I guess this means that consumer auth provider no longer depends on any of our package. We may want to reevaluate what we want to have in cryptography package as it is no longer used by service layer.